peterjohnson/PETER_JOHNSON.md

```diff
@@ -0,0 +1,77 @@
+Whether you want to be a SDET, manager or software engineer find out what skills you lack or are weak in and create a 
+plan to work towards improving them. Ask for feedback on your work and skills and be prepared to actively listen to 
+other opinions, both positive and constructive feedback. Even when something seemingly went well, ask for feedback! 
+You can then form a plan on how to improve, for example I remember giving a presentation to an engineering team on a 
+python testing tool, the presentation was well-received by the audience by the comments I received, but when I asked a 
+few close colleagues on feedback, they mentioned I went a little too fast for some members of my audience.
+Face a weakness with curiosity to make it a strength. When I stumble on a failure, I see it as a bump in the road that I
+will overcome as I have become aware of an area that I can improve.
+
+## Find a mentor
+Find a mentor in a skill you want to develop, for example I recently found a squash coach to help me develop as a squash
+player and to develop my skills and techniques on the court, this is not only helping towards my health goals, but I 
+have been able to benefit from somebody who has more experience than me, which is extremely valuable. Throughout my 
+career I have never directly had a single mentor, but I have worked with individuals who I could learn from and 
+respected their knowledge and opinions. When somebody is willing to spend their own time to teach you something and is 
+honest on what they know and don’t know, have gratitude to know and work with such people as they will enable you to 
+develop and grow. In general, try to surround yourself with people who have similar goals or aspirations, so that you 
+can share experiences and hold each other accountable.
+
+## Work when nobody is looking
+You don’t need to tell everyone your plan or long-term goals in order to succeed in them. Your goals are personal to 
+your values and beliefs, so you don’t require them to be socially approved by your peers or colleagues. 
+This is important as everyone has different aspirations and opinions that can muddy your thoughts. 
+Find someone who can hold you accountable for tasks towards your short-term goals, such as asking you ‘did you write 
+that blog post last week?’ or ‘how did your LeetCode challenges go last night?’, this will help you achieve momentum 
+towards your goals.
+
+## Align your career with your superpower to go that extra mile
+If you find you are naturally enjoying one skill more than another, why not consider a role that allows you to use your 
+strongest skill the most. For example, if you enjoy communicating and leadership more than coding then maybe working as 
+a SDET may not be a good fit as it’s very engineering focused but working as a QA engineer allows you to utilise your 
+communication skills to influence the overall testing strategy for example, and you can end up in a QA lead position.
```

Comment:
Consider adding an 'About the Author' section with your name as a title and where people can find more information about you. James Thomas has one in his PR.